docs/source/tutorial/basic.rst

```diff
@@ -17,7 +17,7 @@ The :class:`cupy.ndarray` class is in its core, which is a compatible GPU altern
 
```

Comment:
We need the description like "In the following code, `cp` is an abbreviation of `cupy`, as `np` is `numpy` as with customarily done.